tools/online_editor/src/shared/properties.ts

```diff
@@ -284,11 +310,26 @@ export class PropertiesView {
                         );
                     }
                 });
+
+                delete_button.className = this.#remover_icon_class;
+                if (element_range != null) {
+                    delete_button.addEventListener("click", () => {
```

Comment:
I wonder if the closure should also take the event argument and call at least stopPropagation() on it.